kafka-ui-react-app/src/components/Topics/List/List.tsx

```diff
@@ -41,25 +44,38 @@ const List: React.FC<Props> = ({
   const { isReadOnly } = React.useContext(ClusterContext);
   const { clusterName } = useParams<{ clusterName: ClusterName }>();
   const { page, perPage } = usePagination();
+  const [orderBy, setOrderBy] = React.useState<TopicColumnsToSort | null>(null);
+  const [search, setSearch] = React.useState<string>('');
```

Comment:
Hmm..., it looks like 'search' and 'order' should be a part of redux store. Because both affect topic list. And it would be great to understand what filter order params were applied